content/en/logs/log_collection/android.md

```diff
@@ -81,216 +46,27 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
                         clientToken = <CLIENT_TOKEN>,
                         env = <ENV_NAME>,
                         variant = <APP_VARIANT_NAME>
-                    )
-                    .useSite(DatadogSite.EU1)
+                    ){{< region-param key=android_logs_sdk_site_kotlin >}}
```

Comment:
Sorry I didn't catch this the first time, can you also preserve indentation here too? So that both methods are lined up (non blocking, it's more for readability)

```
.useSite(DatadogSite.EU1)
.build()
```